### ISSUE TITLE
Bug in cluster points in choropleth

### ISSUE THREAD

**Issue by thibautjombart** [reconhub/gisfirstaid#12]
In the section **Plot cases as clustered points**: when drilling all the way down to individual cases, the cases themselves do not display, instead the 'missing icon' images are used.

Bug in the original package or in our use of it?

@jpolonsky @dirkschumacher

**Comment by jpolonsky** [comment]
yeah that's weird - it's taken basically verbatim from the demo at https://rstudio.github.io/leaflet/markers.html which does resolve to markers. Not sure what's going on here...
